common/lc_shortcuts.cpp

```diff
@@ -58,7 +58,7 @@ void lcResetDefaultMouseShortcuts()
 void lcKeyboardShortcuts::Reset()
 {
 	for (int CommandIdx = 0; CommandIdx < LC_NUM_COMMANDS; CommandIdx++)
-		mShortcuts[CommandIdx] = qApp->translate("Shortcut", gCommands[CommandIdx].DefaultShortcut);
```

Comment:
I think the original code is correct, if you remove this then shortcuts can't be different for different languages. I found that changing the german translation file to be Ctrl+ instead of Strg+ works.

Reply:
Doing the translation here is not necessary, because QT does the translation, since we also imported their translators. Pls see changes in qtmain.cpp too.

Reply:
This isn't related to Qt translations, they don't know how to translate our shortcuts.  For example, the shortcut for the Front view is F in English, but it could be translated to V in German for Vorne in leocad_de.ts .

It doesn't look like any shortcuts have been translated so this change won't break anything, but if the ability to translate shortcuts is being removed then the right thing to do would be to remove the translation markers for shortcuts from lc_commands.cpp as well.

My computer is always set to English so I don't know what the expected behavior for other languages is. I've tested this a bit and it doesn't look like any apps translate shortcuts, even though the words are different. Ctrl+O is always Open, even in Portuguese where the action is Abrir and doesn't have an O at all. Are shortcuts ever translated?

Reply:
You're right the shortcuts are not translated with the language, but the key names are. So Ctrl+O always remains Ctrl+O and only the involved keys are translated. So Ctrl+O becomes Strg+O in German.

I also agree that the translation markers from the lc_commands.cpp can be removed as well. I'll do it.